### DISCUSSION TITLE
Additional relays/sensors on Esp devices

### DISCUSSION THREAD

**Discussion by bowlerhatman** [BernhardSchlegel/BierBot-Bricks#33] [Ideas & feature wishlist]
Hi there! Has anyone looked into implementing the bricks on esp8266 but adding an extra ds18b20 or two, and/or some extra relays? Especially for a fermentation controller a second channel would be very useful.

**Comment by JoDaNl** [comment]
Hi,

Not on a esp8266 but for esp32 I developing a fermentation controller..but this work in progress:

https://github.com/JoDaNl/BookesBrick



-------- Oorspronkelijk bericht --------
Van: bowlerhatman ***@***.***>
Datum: 04-04-2023 10:58 (GMT+01:00)
Aan: BernhardSchlegel/BierBot-Bricks ***@***.***>
Cc: Subscribed ***@***.***>
Onderwerp: [BernhardSchlegel/BierBot-Bricks] Additional relays/sensors on Esp devices (Discussion #33)


Hi there! Has anyone looked into implementing the bricks on esp8266 but adding an extra ds18b20 or two, and/or some extra relays? Especially for a fermentation controller a second channel would be very useful.

**Reply by bowlerhatman** [reply]
That looks cool! Is there any particular reason it wouldn't work on an ESP8266 as well?

**Comment by JoDaNl** [comment]
Can't think of any particular reason...just targetted to the ESP32 has I had these on stock...
So this could well work. Cannot try on short notice due to some traveling I'm doing.



Verzonden vanaf mijn Galaxy



-------- Oorspronkelijk bericht --------
Van: bowlerhatman ***@***.***>
Datum: 04-04-2023 12:17 (GMT+01:00)
Aan: BernhardSchlegel/BierBot-Bricks ***@***.***>
Cc: JoDaNl ***@***.***>, Comment ***@***.***>
Onderwerp: Re: [BernhardSchlegel/BierBot-Bricks] Additional relays/sensors on Esp devices (Discussion #33)


That looks cool! Is there any particular reason it wouldn't work on an ESP8266 as well?

**Reply by bowlerhatman** [reply]
Awesome, when my 8266s arrive I'll try it out!

**Comment by BernhardSchlegel** [comment]
Hi there, aside from the awesome BookesBrick you can also checkout the Raspberry Pi Implementation which will ask you for the number of temperature sensors and relays at setup https://github.com/BernhardSchlegel/BierBot-Bricks-RaspberryPi

**Reply by bowlerhatman** [reply]
Thanks Bernhard, I'm aware of the pi version, my plan is to use the pi on the hot side, and then I'm hoping for just a single additional channel for the fermentation controller, which doesn't need that much more complexity. The cost of pis at the moment makes them overkill for such a limited expansion of capabilities. Definitely still an option though!
